downloads/swan-lake-release-notes/swan-lake-2201.9.0/RELEASE_NOTE.md

```diff
@@ -60,7 +60,46 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 9 (2201.9.0)](
 
 #### Language Server
 
-#### CLI
+#### Test Framework
+
+- The test framework now supports for parallel execution of tests as an experimental feature. To activate the parallel test execution, simply use the `--parallel` flag with the `bal test` command.
```

Comment:
Don't we need to mention it as an experimental feature? https://github.com/ballerina-platform/ballerina-dev-website/blob/750cbc1f5e6ae3537754e7b48d4a12a0c7a67021/downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md?plain=1#L186

Reply:
We have included that it is experimental.